### PR TITLE
Make the license page extendable

### PR DESCRIPTION
**Pull request by emanuelen5**
This license page can be configurable per Makerspace without adding any code to git

### CODE REVIEW

File: public/scss/style.scss [hunk 1]
```diff
@@ -2026,3 +2026,7 @@ $palette-red: $palette4;
         pointer-events: none;
     }
 }
+
+code {
+    white-space: preserve !important;
```
Comment (HalfVoxel) [inline public/scss/style.scss:2031]: `!important` is best to avoid imo.

File: api/src/member/member.py [hunk 2]
```diff
@@ -75,3 +77,26 @@ def get_member_groups(member_id: int) -> List[Any]:
...
+    readme_path = repo_root / "data/licenses/README.md"
+    html_files = list(repo_root.glob("data/licenses/*.html"))
+    md_files = list(repo_root.glob("data/licenses/*.md"))
+
+    no_licenses_added = (html_files + md_files) == [readme_path]
+    if no_licenses_added:
+        return _get_file_content_as_html(readme_path)
+
+    all_files_to_render = [f for f in html_files + md_files if f != readme_path]
+    combined_file_content = [_get_file_content_as_html(f) for f in all_files_to_render]
```
Comment (HalfVoxel) [inline api/src/member/member.py:100]:
```suggestion
    all_files_to_render = [f for f in html_files + md_files if f != readme_path]
    if len(all_files_to_render) == 0:
        all_files_to_render = [readme_path]

    combined_file_content = [_get_file_content_as_html(f) for f in all_files_to_render]
```

**Review by HalfVoxel — APPROVED** [review]

File: public/scss/style.scss (hunk 1 above)
Comment (emanuelen5) [reply public/scss/style.scss:2031]: Agreed. In this case it doesn't work without it since the rule is already set by UIkit.

Comment (HalfVoxel) [reply public/scss/style.scss:2031]: You can at least make it more specific? If a rule is more specific than another rule, it overrides it anyway.

**Pull request merged**